src/composables/pools/usePoolCreation.ts

```diff
@@ -0,0 +1,131 @@
+import { balancerSubgraphService } from '@/services/balancer/subgraph/balancer-subgraph.service';
```

Comment:
Again, would be good to group and structure composables in the same way I proposed for components so that it's easier to parse.